managed/services/server/updater.go

```diff
@@ -40,13 +41,12 @@ import (
 
 // defaultLatestPMMImage is the default image name to use when the latest version cannot be determined.
 const (
-	defaultLatestPMMImage        = "perconalab/pmm-server:3-dev-latest"
-	pmmUpdatePerformLog          = "/srv/logs/pmm-update-perform-init.log"
-	defaultVersionServiceAddress = "https://check-dev.percona.com"
-	updateCheckInterval          = 24 * time.Hour
-	updateCheckResultFresh       = updateCheckInterval + 10*time.Minute
-	updateDefaultTimeout         = 30 * time.Second
-	envfilePath                  = "/home/pmm/update/pmm-server.env"
+	defaultLatestPMMImage  = "perconalab/pmm-server:3-dev-latest"
```

Comment:
I wonder if we should update this now to a stable version or wait till GA is released.

Reply:
is it still used somewhere in code?

Reply:
No even, dropped it now.